Doc/library/dis.rst

```diff
@@ -24,6 +24,12 @@ interpreter.
       Use 2 bytes for each instruction. Previously the number of bytes varied
       by instruction.
 
+   .. versionchanged:: 3.11
```

Comment:
Maybe don't document this until we are sure it's what we want?

Reply:
I'd prefer to keep it, and change it if/when `dis` changes (which is easy enough). That way we don't forget to document it.

Reply:
OK. Once we've started doing this we will need to complete it before the beta release anyway.